core/src/main/java/com/tapptitude/core/model/extension/ImageExt.kt

```diff
@@ -2,7 +2,10 @@ package com.tapptitude.core.model.extension
 
 import com.tapptitude.core.model.Image
 import com.tapptitude.network.dto.ImageDto
+import com.tapptitude.persistence.images.ImageEntity
 
-fun ImageDto.toImage() = Image(imageUrl = imageUrl ?: "")
+fun ImageDto.toEntity() = ImageEntity(url = imageUrl.orEmpty())
```

Comment:
Can't we make them internal? We are exposing `Image` to the outside user, we can keep `ImageDto` and `ImageEntity` to ourselves